### ISSUE TITLE
[3rd] Tradução Cap. 2 - "Program Structure"

### ISSUE THREAD

**Issue by joaostein** [braziljs/eloquente-javascript#401]
Responsável: @rands0n

**Comment by joaostein** [comment]
@rands0n, estou enviando essa mensagem para todos os responsáveis apenas para checar se está tudo certo com a sua tradução.  Se precisar de qualquer coisa, só avisar.

_ps. A data da [milestone](https://github.com/braziljs/eloquente-javascript/milestone/2) para finalizar a tradução está para 24 de Dezembro (ho-ho 🎅🎁).  Se você achar que não for realista, só avisar, ok?_

**Comment by joaostein** [comment]
@rands0n - poderia dar um update e qual o seu plano para tradução deste capítulo?

**Comment by marceloagil** [comment]
Oi @rands0n , tudo bom? Eu gostaria de traduzir este capítulo. Posso?

**Comment by joaostein** [comment]
@marceloagil vamos começar com um capítulo por vez. assim que finalizar o #399 eu lhe entrego este, pode ser?

**Comment by GSFZamai** [comment]
Bom dia, 
@rands0n, @joaostein 
gostaria de contribuir com a tradução. Ainda há vagas disponíveis?

**Comment by joaostein** [comment]
@GSFZamai claro, coloquei você como responsável deste capítulo. Se precisar de qualquer coisa, é só me avisar.

**Comment by GSFZamai** [comment]
@joaostein ok, obrigado, um abraço!

**Comment by joaostein** [comment]
Encerrando — pulamos a 3ª edição e fomos direto pra 4ª (#490).
